docs/UserGuide.md

```diff
@@ -32,80 +32,173 @@
 > **Command Format**
 > * Words in `UPPER_CASE` are the parameters.
 > * Items in `SQUARE_BRACKETS` are optional.
-> * Items with `...` after them can have multiple instances.
-> * The order of parameters is fixed.
+> * The order of parameters is fixed, with optional parameters being the exception.
+> * Parameters `START_DATE` and `END_DATE` has to be in dd-mm-yyyy format (e.g. 23-10-2016).
+> * Parameters `START_TIME` and `END_TIME` has to be in 24 hour format (e.g. 1215).
+> * Parameter `TIMELEFT` **must be a positive integer**, indicating number of hours left.
 
 #### Viewing help : `help`
 Format: `help`
 
 > Help is also shown if you enter an incorrect command e.g. `abcd`
  
-#### Adding a person: `add`
-Adds a person to the address book<br>
-Format: `add NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]...` 
+#### Adding a task : `add`
+Adds a task to the task manager.<br>
+Format: `add TASK_NAME [sd/START_DATE] [st/START_TIME] [ed/END_DATE] [et/END_TIME] [tl/TIMELEFT]` 
```

Comment:
if you plan to use any of the date-time parsers, may be you can do away with one of the fields say, time and use just one delimiter and let the parser handle the rest?